buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -265,41 +265,34 @@ def _build_with_inject(self,
 
     # pylint: disable=too-many-arguments
     @retry(python_on_whales.exceptions.DockerException,
-           tries=5,
+           tries=3,
            delay=1,
            backoff=3,
-           max_delay=30,
+           max_delay=10,
            logger=logger)
     def _build_single_image(self,
                             name: str,
                             platform: str,
-                            push: bool = True,
-                            path: str = ".",
-                            file: str = "Dockerfile",
-                            tags: List[str] = None,
-                            build_args: dict = None,
-                            built_images: list = None,
-                            inject: dict = None,) -> None:
+                            push: bool,
+                            path: str,
+                            file: str,
+                            tags: List[str],
+                            build_args: dict,
+                            mp_image_name: str,
+                            inject: dict) -> None:
         """
         Builds a single image for the given platform
 
         Args:
             name (str): The name of the image
             platform (str): The platform to build the image for (e.g. linux/amd64)
-            push (bool, optional): Whether to push the image to the registry. Defaults to True.
-            path (str, optional): The path to the Dockerfile. Defaults to ".".
-            file (str, optional): The path/name of the Dockerfile (ie. <path>/Dockerfile). Defaults to "Dockerfile".
-            tags (List[str], optional): The tags to apply to the image. Defaults to None.
-            docker_registry (str, optional): The docker registry to push the image to. Defaults to None.
-            built_images (list, optional): A list of built images. Defaults to None.
+            push (bool): Whether to push the image to the registry.
+            path (str): The path to the Dockerfile.
+            file (str): The path/name of the Dockerfile (ie. <path>/Dockerfile).
+            tags (List[str]): The tags to apply to the image.
+            docker_registry (str): The docker registry to push the image to.
+            mp_image_name (str): A list of built images.
```

Comment:
How is a single string a list of images?

Reply:
Fixed